src/test_solvers/solvers.cu

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+
+#include "solvers.h"
+#include <cusp/monitor.h>
+#include <cusp/print.h>
+#include <cusp/krylov/gmres.h>
+
+#include <cuda_runtime.h>
+// where to perform the computation
+typedef cusp::device_memory MemorySpace;
+
+std::vector<float> cudaSolver(std::vector<int>* rowInd, std::vector<int>* colInd, std::vector<float>* val, std::vector<float>* b){
```

Comment:
Try to format lines so that they don't exceed 80 characters (yes, the 1970ies still live on :)

Reply:
Oh. I will clang-format all files!